### ISSUE TITLE
[MEMO] Preference Optimization, Multi-Objective Alignment, and Selective Prediction

### ISSUE THREAD

**Issue by inoue0426** [inoue0426/Paper-reading#138]
# Preference Optimization and RLHF

- Direct Preference Optimization: Your Language Model is Secretly a Reward Model
  - https://openreview.net/forum?id=HPuSIXJaa9

- Training Language Models to Follow Instructions with Human Feedback
  - https://arxiv.org/abs/2203.02155

- Deep Reinforcement Learning from Human Preferences
  - https://proceedings.neurips.cc/paper/2017/hash/d5e2c0adad503c91f91df240d0cd4e49-Abstract.html

- Learning to Summarize from Human Feedback
  - https://arxiv.org/abs/2009.01325

- Constitutional AI: Harmlessness from AI Feedback
  - https://arxiv.org/abs/2212.08073

- A General Theoretical Paradigm to Understand Learning from Human Preferences
  - https://arxiv.org/abs/2310.12036

- KTO: Model Alignment as Prospect Theoretic Optimization
  - https://arxiv.org/abs/2402.01306

- ORPO: Monolithic Preference Optimization without Reference Model
  - https://aclanthology.org/2024.emnlp-main.626/

- SimPO: Simple Preference Optimization with a Reference-Free Reward
  - https://arxiv.org/abs/2405.14734

- Provably Robust DPO: Aligning Language Models with Noisy Feedback
  - https://openreview.net/forum?id=yhpDKSw7yA

# Multi-Objective Alignment and Pareto Optimization

- A Survey of Multi-Objective Sequential Decision-Making
  - https://arxiv.org/abs/1402.0590

- A Practical Guide to Multi-Objective Reinforcement Learning and Planning
  - https://link.springer.com/article/10.1007/s10458-022-09552-y

- Rewarded Soups: Towards Pareto-Optimal Alignment by Interpolating Weights Fine-Tuned on Diverse Rewards
  - https://arxiv.org/abs/2306.04488

- Beyond One-Preference-Fits-All Alignment: Multi-Objective Direct Preference Optimization
  - https://arxiv.org/abs/2310.03708

- Rewards-in-Context: Multi-Objective Alignment of Foundation Models with Dynamic Preference Adjustment
  - https://proceedings.mlr.press/v235/yang24q.html

- Controllable Preference Optimization: Toward Controllable Multi-Objective Alignment
  - https://arxiv.org/abs/2402.19085

- Personalized Soups: Personalized Large Language Model Alignment via Post-hoc Parameter Merging
  - https://arxiv.org/abs/2310.11564

- SteerLM: Attribute Conditioned SFT as an User-Steerable Alternative to RLHF
  - https://arxiv.org/abs/2310.05344

- HelpSteer: Multi-attribute Helpfulness Dataset for SteerLM
  - https://aclanthology.org/2024.naacl-long.185/

- Pareto Conditioned Networks
  - https://arxiv.org/abs/2204.05036

# Selective Prediction, Abstention, and Calibration

- On the Foundations of Noise-Free Selective Classification
  - https://jmlr.org/papers/v11/el-yaniv10a.html

- Selective Classification for Deep Neural Networks
  - https://proceedings.neurips.cc/paper/2017/hash/4a8423d5e91fda00bb7e46540e2b0cf1-Abstract.html

- SelectiveNet: A Deep Neural Network with an Integrated Reject Option
  - https://proceedings.mlr.press/v97/geifman19a.html

- Deep Gamblers: Learning to Abstain with Portfolio Theory
  - https://arxiv.org/abs/1907.00208

- On Calibration of Modern Neural Networks
  - https://proceedings.mlr.press/v70/guo17a.html

- Language Models (Mostly) Know What They Know
  - https://www.anthropic.com/research/language-models-mostly-know-what-they-know

- Just Ask for Calibration: Strategies for Eliciting Calibrated Confidence Scores from Language Models Fine-Tuned with Human Feedback
  - https://arxiv.org/abs/2305.14975

- Detecting Hallucinations in Large Language Models Using Semantic Entropy
  - https://www.nature.com/articles/s41586-024-07421-0

- SelfCheckGPT: Zero-Resource Black-Box Hallucination Detection for Generative Large Language Models
  - https://aclanthology.org/2023.emnlp-main.557/

- Do LLMs Know When to NOT Answer? Investigating Abstention Abilities of Large Language Models
  - https://arxiv.org/abs/2407.16221
